packages/zowe-explorer/src/trees/dataset/DatasetActions.ts

```diff
@@ -1240,24 +1240,24 @@ export class DatasetActions {
                 const job = await ZoweExplorerApiRegister.getJesApi(sessProfile).submitJcl(doc.getText());
                 const args = [sessProfileName, job.jobid];
                 const setJobCmd = `${Constants.SET_JOB_SPOOL_COMMAND}?${encodeURIComponent(JSON.stringify(args))}`;
-                const notifyButton = vscode.l10n.t("Start Polling");
+                const notifyButton = vscode.l10n.t("Poll For Job Completion");
                 const jobDisplay = `${job.jobname}(${job.jobid})`;
                 const message = vscode.l10n.t({
-                    message: "Job submitted {0}",
+                    message: "Job submitted: {0}",
                     args: [`[${jobDisplay}](${setJobCmd})`],
                     comment: ["Job name and ID with set job command"],
                 });
-                const selection = await Gui.showMessage(message, { items: [notifyButton] });
-                if (selection === notifyButton) {
-                    DatasetActions.pollSubmittedJob(sessProfile, sessProfileName, job.jobid, job.jobname);
-                }
                 ZoweLogger.info(
                     vscode.l10n.t({
                         message: "Job submitted {0} using profile {1}.",
                         args: [job.jobid, sessProfileName],
                         comment: ["Job ID", "Profile name"],
                     })
                 );
+                const selection = await Gui.showMessage(message, { items: [notifyButton] });
+                if (selection === notifyButton) {
+                    DatasetActions.pollSubmittedJob(sessProfile, sessProfileName, job.jobid, job.jobname);
+                }
```

Comment:
(nitpick) The current code blocks this function until the user interacts with the notification. Since the job has already been submitted at this point, consider using `.then()` instead of `await` for the `Gui.showMessage` call - this allows the function to return immediately while still handling the user's response if they click the button. The same applies to the `submitMember` change on line 1499.